src_rebuild/Game/C/wheelforces.c

```diff
@@ -528,6 +528,32 @@ void AddWheelForcesDriver1(CAR_DATA* cp, CAR_LOCALS* cl)
 	else
 	{
 		cp->hd.wheel_speed = cdz / 64 * (cl->vel[2] / 64) + cdx / 64 * (cl->vel[0] / 64);
+
+		// [A] wibbly wobbly fuckery hack...
+		if (car_cos->extraInfo & 4)
```

Comment:
@SoapyMan Do you understand the physics enough to maybe understand why the box truck in Vegas needs this hack? It usually happens when you drive a bit, then come to a hard stop using the brake + handbrake, and exit the vehicle. Very consistently, it will jitter like an absolute maniac. Seems to be related to how angular velocity is calculated? While this bug is happening, the car will stay at a constant speed of about 3, and the wheel speed goes all over the damn place.

The only way I've managed to get it to stop is to manually bring the acceleration/velocity values down manually. But I haven't found any other car that does this yet.. so WTF is going on?! It's driving me nuts! I hate hacks and would really like to fix this another way if possible.

Reply:
Also, this technically resolves a bug where the trucks in Destroy the Yard may end up phasing out of the world by the time you arrive.. I was like WTF when I noticed it happen the first time!